R/post_destroy.R

```diff
@@ -0,0 +1,19 @@
+#' Delete status of user's Twitter account
+#'
+#' Deletes a status of user's profile.
+#' @inheritParams post_tweet
+#' @export
+#' @examples
+#' \dontrun{
+#' pt <- post_tweet()
+#' crt <- httr::content(pt)
+#' post_destroy(crt$id_str)
+#' }
+post_destroy <- function(destroy_id, token = NULL) {
```

Comment:
We should probably briefly contemplate whether this would be better called `tweet-destroy()` or `status_destroy()` — I think the existing `post_` prefix confusingly refers to the HTTP method, not a twitter "post" (i.e. a tweet).

Reply:
We are discussing this on #480